baitap-19-10-2020/bai-2.js

```diff
@@ -0,0 +1,10 @@
+const yearOld = (age) => {
+    if(age >= 16 )
+    {
+        console.log ('Học sinh đủ điều kiện vào lớp 10')
+    } 
+    else
+    {
+        console.log('Học sinh không đủ điều kiện vào lớp 10')
+    } 

```

Comment:
```suggestion
    if(age >= 16 ) return true;
    else return false;
```
Hoặc:

```suggestion
    return age >= 16;
```

Đơn giản chỉ cần return về boolean là đc